ambry-api/src/main/java/com.github.ambry/account/ContainerBlobsResource.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package com.github.ambry.account;
+
+/**
+ * An {@link AclService.Resource} that represents the blobs in a {@link Container}.
+ *
+ */
+class ContainerBlobsResource implements AclService.Resource {
```

Comment:
Because we envision possibly having multiple resources for the container. For example, there can be the blobs in the container (what this resource is for) and another resource for container metadata (capacity, description, name, etc.). This change was made because we simplified the operations in the `Operation` enum to be just verbs instead of verb-noun combos.